dist/tools/drone-scripts/build_and_test.sh

```diff
@@ -19,27 +32,32 @@ export TRAVIS_BUILD_NUMBER="${CI_BUILD_NUMBER}"
 export TRAVIS_COMMIT="${DRONE_COMMIT}"
 export TRAVIS_BUILD_DIR="${DRONE_BUILD_DIR}"
 
-for group in static-tests \
-             avr8         \
-             msp430       \
-             x86          \
-             arm7         \
-             cortex_m0    \
-             cortex_m3_1  \
-             cortex_m3_2  \
-             cortex_m4    \
-    ;
-do
-    echo "Begin group ${group}"
-    BUILDTEST_MCU_GROUP=${group} ./dist/tools/travis-scripts/build_and_test.sh "$@"
-    RES=$?
-    echo "Result (${group}): ${RES}"
-    if [ ${RES} -ne 0 ]; then
-        FAILURES=$((${FAILURES} + 1))
-    fi
-    SUMMARY_MESSAGE="${SUMMARY_MESSAGE}${group}: ${RES}\n"
+export -f exec_build_func
+
+TMPDIR=`mktemp -d`
```

Comment:
Did I overlook something? `MYTMPDIR` in the `trap` vs. `TMPDIR` elsewhere. I suggest you change `TMPDIR` to `MYTMPDIR` across the script